metrics/lib/json.bash

```diff
@@ -129,6 +131,43 @@ metrics_json_end_of_system() {
 	system_index=$(( ${#json_result_array[@]}))
 }
 
+# Add a array of number of physical sockets and hypervisor per node within the Kubernetes cluster
+metrics_json_num_sockets_node(){
+
+	metrics_json_start_nested_array
```

Comment:
This actually isn't a nested array, but just an array. change to `metrics_json_start_array`. Right now this is getting put in the BootResults array rather the `metadata` object.

Reply:
Done.